packages/web-components/src/components/va-textarea/va-textarea.scss

```diff
@@ -6,6 +6,7 @@
 @use 'usa-error-message/src/styles/usa-error-message';
 @use 'usa-hint/src/styles/usa-hint';
 @use 'usa-textarea/src/styles/usa-textarea';
+@use 'usa-character-count/src/styles/_usa-character-count';
```

Comment:
Oh cool there's a uswds package for this! 👍

Reply:
@it-harrison I just noticed there is a rem value in that class. Can you update the formations override stylesheet so that it converts that to px? We are having to do this temporarily while Formation is still the base stylesheet:

```css
.usa-character-count__status {
    padding-top: 0.25rem;
}
```

![Screenshot 2023-07-21 at 12 45 46 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/c83ce263-0f7e-43db-812f-f56198127912)